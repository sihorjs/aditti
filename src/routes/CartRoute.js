import React from "react";
import Cart from "components/Cart";

function CartRoute() {
    return <Cart />;
}

export default CartRoute;
