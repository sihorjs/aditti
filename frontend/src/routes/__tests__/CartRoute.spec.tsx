import React from "react";
import { shallow } from "enzyme";
import CartRoute from "routes/CartRoute";

describe("<CartRoute />", () => {
    it("renders without crash", () => {
        const tree = shallow(<CartRoute />);

        expect(tree).toMatchSnapshot();
    });
});
