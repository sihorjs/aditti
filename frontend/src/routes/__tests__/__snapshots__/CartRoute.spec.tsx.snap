// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartRoute /> renders without crash 1`] = `ShallowWrapper {}`;
